### ISSUE TITLE
bugstastic stuff

### ISSUE THREAD

**Issue by Lovecutehuy1943** [TrashboxBobylev/Rat-King-Adventure#28]
these are already (probably) reported but I will report them here again as a reminder
1/ drratedon talent at +4 doesnt work properly with ratforcements
either it spawns the rats without the champion titles or it crashes the game
2/ kromer economy doesnt work properly in the abyss
the enemies in the abyss still get the adrenaline buff and they still spawn constantly but no kromers will appear
3/ forbidden runes challenge isnt working at all
I still get 15 SoU during the run, basically making the challenge free score
4/ rat magic's shielding when zapping at full charge doesnt give any shielding
5/ the abyssal dragons (not warped) when they reach 0 hp goes into a state of being both alive and dead
basically, they cant die but they can only see 1 tile around them
(im not sure how this is possible, I will pay a closer attention and edit this message after I did enough testing)
